mikeio/spatial/_FM_geometry.py

```diff
@@ -970,10 +969,7 @@ def isel(
         find_index : find element indicies for points or an area
         """
 
-        if self._type == DfsuFileType.DfsuSpectral1D:
-            return self._nodes_to_geometry(nodes=idx)
```

Comment:
Why is this no longer necessary?

Reply:
Since LineSpectrum has a dedicated geometry class https://github.com/DHI/mikeio/blob/b8d472713f5223e9d2f4ce01c1596a92e533ea51/mikeio/spatial/_FM_geometry_spectral.py#L179-L181